modules/gov/gov_params.go

```diff
@@ -0,0 +1,83 @@
+package gov
+
+import (
+	"encoding/json"
+	"fmt"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/params"
+)
+
+var DepositProcedureParameter DepositProcedureParam
+
+type DepositProcedureParam struct {
+	Value DepositProcedure
+	ps    params.Setter
+	pg    params.Getter
+}
+
+func (param *DepositProcedureParam) InitGenesis(genesisState interface{}) {
+	if value, ok := genesisState.(DepositProcedure); ok {
```

Comment:
What happens if the genesisState is nil？

Reply:
You can see the UnitTest. The value of `ok` is `false`.